README.md

```diff
@@ -0,0 +1,52 @@
+# service-control
+
+| Branch | Status |
+|--------|-----------|
+| develop | ![Build Status](https://codebuild.eu-central-1.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiZk43RFNPWm5aNldBYmE3NU95MkFWNVg0aG5oMk1VRlhVcHNmdEVGOGFwc05zRW1lVG4zaU40dnBtSUFsd2dxd0tESlNYN1VkSS9pbkpFWDJ1ajQ0dkhrPSIsIml2UGFyYW1ldGVyU3BlYyI6IktHVHNJL21aN0NKKzg0V2YiLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=develop) |
```

Comment:
I guess you copied the correct badge from the Codebuild project ?

Reply:
The one from `service-control-release` because I figured it can't be `service-control-pr` because it's develop and master :thinking: